src/registration/ApplicationActivation.Library/ApplicationActivationService.cs

```diff
@@ -89,8 +89,8 @@ private async Task<IApplicationChecklistService.WorkerChecklistProcessStepExecut
         {
             throw new ConflictException($"CompanyApplication {context.ApplicationId} is not in status SUBMITTED");
         }
+        var (companyId, companyName, businessPartnerNumber, iamIdp, applicationTypeId, networkRegistrationProcessId) = result;
```

Comment:
please change to iamIdps since it's an enumerable

```suggestion
        var (companyId, companyName, businessPartnerNumber, iamIdps, applicationTypeId, networkRegistrationProcessId) = result;
```